src/__fixtures__/Events/24-special_special.json

```diff
@@ -1,8 +1,72 @@
 {
     "name": "Special special tags",
     "options": {},
-    "html": "<sCriPT></scripter</soo</sCript><STyLE></styler</STylE><sCiPt><stylee><scriptee><soo>",
+    "html": "<tItLe><b>foo</b><title></TiTlE><sitle><b></b></sitle><ttyle><b></b></ttyle><sCriPT></scripter</soo</sCript><STyLE></styler</STylE><sCiPt><stylee><scriptee><soo>",
```

Comment:
The `<sitle>` and `<ttyle>` tests are to make sure that logic for the specials doesn't get refactored to something that allows the checking of a `t` or and `s` arbitrarily followed by the other specials logic (I almost implemented this very same error, so I figured a test to protect against it would be useful)